.buildkite/pipeline.yml

```diff
@@ -84,26 +85,27 @@ steps:
           manual:
             allowed: true
 
-      # Runs inly on the main branch
-      - label: "Unit tests - MacOS 13 ARM"
-        key: "unit-tests-macos-13-arm"
+  - group: "MacOS tests"
```

Comment:
The official naming is now `macOS`, which we adopted in the beats repo. Might as well update it here too.

Reply:
Renamed